include/config/rcxml.h

```diff
@@ -141,6 +142,7 @@ struct rcxml {
 		bool show;
 		bool preview;
 		bool outlines;
+		bool all_workspaces;
```

Comment:
We should not have a dedicated `all_workspace` bool here. It's already covered by one of the bits in `criteria`. Just do a bitwise check.